CODEOWNERS

```diff
@@ -8,3 +8,7 @@
 
 # Prevent PRs that only modify package.json from notifying everyone
 package.json
+
+# Owned by data platform
+src/resources/AnalyticsAdmin/ @coveo/dpf-ui-owners-1
+src/resources/UsageAnalytics/ @coveo/dpf-ui-owners-1
```

Comment:
Would someone be able to give https://github.com/orgs/coveo/teams/dpf-ui-owners-1/ write access to the repository, if they have a moment?

Reply:
Done!